sympy/matrices/eigen.py

```diff
@@ -838,16 +836,16 @@ def _is_positive_semidefinite_minors(M):
 
 
 def _jordan_form(M, calc_transform=True, **kwargs):
-    """Return ``(P, J)`` where `J` is a Jordan block
-    matrix and `P` is a matrix such that
+    """Return $(P, J)$ where $J$ is a Jordan block
+    matrix and $P$ is a matrix such that
 
-        ``M == P*J*P**-1``
+        $M == P*J*P**-1$
```

Comment:
This needs to be reformatted for LaTeX.

Reply:
Also if it's going to be on its own line, it should use display math

```
.. math::
   M = ...
```

Reply:
How's ``$M = P J P^{-1}$`` for this - or do you use an explicit symbol for matrix multiplication?

(I am inclined to put it inline with the text)

Reply:
That looks fine to me. It matches what the other docstrings use.